docs/components/menus.md

```diff
@@ -0,0 +1,353 @@
+<!--docs:
+title: "Menus"
+layout: detail
+section: components
+excerpt: "Menus display a list of choices on temporary surfaces."
+iconId:
+path: /catalog/menus/
+api_doc_root: true
+-->
+
+# Menus
+
+[Menus](https://material.io/components/menus/#) display a list of choices on temporary surfaces.
+
+![Menu hero example showing two cascading menus](assets/menus/menus-hero.png)
+
+# Contents
+
+- [Using menus](#using-menus)
+- [Installing menus](#installing-menus)
+- [Making menus accessible](#making-menus-accessible)
+- [Dropdown menus](#dropdown-menus)
+- [Exposed dropdown menus](#exposed-dropdown-menus)
+- [Theming](#theming)
+
+## Using menus
+
+A menu displays a list of choices on a temporary surface. They appear when users interact with a button, action, or other control.
+
+### Installing menus
+
+### Making menus accessible
+
+Flutter's list component APIs support labeling for accessibility. For more guidance on writing labels, go to [our page on how to write a good accessibility label](https://material.io/design/usability/accessibility.html#writing).
+
+## Anatomy
+
+The following are menu anatomy diagrams showing all possible elements:
+
+![Menus anatomy diagrams](assets/menus/menus-anatomy-composite.png)
+
+1. Container
+1. Leading icon (optional)
+1. Text
+1. Divider (optional)
+1. Command (optional)
+1. Cascading menu indicator (optional)
+1. Selection state (optional)
+
+### Container attributes
+
+&nbsp; | Property
+------ | ---------
+**Color** | Use the `PopupMenuButton` widget. Within this widget use the `Colors` property
+**Height** | Use the `PopupMenuButton` widget. Within this widget use the `padding` property.
+**Overlay window** | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder` property. Each item added to this list will add an additional window.
```

Comment:
This attribute doesn't seem correct. I'm not sure what is meant by OverlayWindow

Reply:
Done